application/views/rss/rss.php

```diff
@@ -42,7 +48,9 @@
       <enclosure url="<?= $section->mp3_64_url?>" length="0" type="audio/mpeg" />
   <itunes:explicit>No</itunes:explicit>
   <itunes:block>No</itunes:block>
-  <itunes:duration><![CDATA[<?= $section->migrated_time?>]]></itunes:duration>
```

Comment:
`migrated_time` sounded familiar, and indeed: https://github.com/LibriVox/librivox-catalog/issues/148

When this lands, I'll just drop that column entirely.